docs/static/includes/interfaces/bicep/int.locks.udt.schema.bicep

```diff
@@ -3,8 +3,9 @@
 //   Parameters   //
 // ============== //
 
+import { lockType } from 'br/public:avm/utl/types/avm-common-types:0.4.0'
```

Comment:
How do we want to maintain this versioning? Are we going to update all schema files every time we update the common type module? Not to necessarily address in this PR, but something to be aligned upon

Reply:
Fair point. We can also replace it with `<version>` as we are doing in the readmes

Reply:
Addressed